test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Converters/IconUrlToImageCacheConverterTests.cs

```diff
@@ -153,6 +153,59 @@ public void Convert_WithValidImageUrl_DownloadsImage()
             }
         }
 
+        [CIOnlyTheory]
+        [InlineData("icon.jpg", "icon.jpg", "icon.jpg", "")]
```

Comment:
As I asked above, what if there are 2+ embedded icons that are not icons? Expect them all to return Default?

Reply:
Yes, both should return the Default icon